### DISCUSSION TITLE
Use rook/ceph image without Kubernetes?

### DISCUSSION THREAD

**Discussion by hrstoyanov** [rook/rook#14438] [Q&A]
Hi, I am new to Ceph. 

Can I sue the rook/ceph docker image without anything Kubernetes? I just just the docker image for development purposes, no meed for real clustering. I only need the S3 API.

Thanks.

**Comment by parth-gr** [comment]
I think you should ceph directly then, https://github.com/ceph/ceph

**Reply by hrstoyanov** [reply]
Do you know of a container image urls? The github ceph project does not mention any.

**Reply by Madhu-1** [reply]
https://github.com/ceph/ceph-container is the link for ceph container image repo

**Reply by travisn** [reply]
@hrstoyanov Note that the two main supported ways to deploy Ceph are [cephadm](https://docs.ceph.com/en/latest/cephadm/) or Rook. Have you looked at cephadm?

**Reply by hrstoyanov** [reply]
I need a way to run Ceph containers for local testing only, what is the best way to do that?

The container repo posted above has not been updated in an year?

Rook seems too K8S biased (I don't use K8S)

**Reply by zhucan** [reply]
https://docs.ceph.com/projects/ceph-ansible/en/stable-8.0/installation/containerized.html @hrstoyanov like this?

**Reply by hrstoyanov** [reply]
Kind of, but just Docker, no Ansible? Simple set up for development in my laptop

**Reply by zhucan** [reply]
https://github.com/ceph/ceph-ansible?tab=readme-ov-file

**Reply by Madhu-1** [reply]
> I need a way to run Ceph containers for local testing only, what is the best way to do that?
> 
> The container repo posted above has not been updated in an year?
> 
> Rook seems too K8S biased (I don't use K8S)

@hrstoyanov did you mean https://github.com/ceph/ceph-container not updated?  or something else.

If its above, its updated and also it pushed the images to quay.io where you can find all the images https://quay.io/repository/ceph/ceph

**Reply by hrstoyanov** [reply]
Ok, so this Ceph base image should have all that is needed for S3 (Rados Gateway) too?
